home_audit.node.js

```diff
@@ -832,21 +835,28 @@ let validationRules = {
     /*
      * hvac_heating
      */
-    heating_type_1: function(value) {
-        if(value === 'none' && _homeValues.cooling_type_1 === 'none') {
-            return new Validation('Heating Type is required if there is no Cooling Type', ERROR);
+    _heating_and_cooling_types: function(value, num, heatingOrCooling) {
+        const opp = heatingOrCooling === HEATING ? COOLING : HEATING;
```

Comment:
I'd really prefer `oppositeSystem` instead of `opp`. You only have to type out the whole name once, code completion will do it for you every other time. And we can afford the bytes :)